rllib/examples/env/greyscale_env.py

```diff
@@ -99,7 +99,7 @@ def env_creator(config):
         lr=0.0001,
         grad_clip=100,
         sgd_minibatch_size=500,
-        train_batch_size=5000,
+        train_batch_size=5000 if not args.as_test else 1000,
```

Comment:
This test will time out for RL Modules otherwise.

Reply:
this is b/c of the tf2 slowness right?